Formula/pacparser.rb

```diff
@@ -0,0 +1,28 @@
+class Pacparser < Formula
+  desc "Library to parse proxy auto-config (PAC) files"
+  homepage "https://github.com/pacparser/pacparser"
+  url "https://github.com/pacparser/pacparser/archive/1.3.6.tar.gz"
```

Comment:
@manugarg np, I need to finish up this PR with the requested tests and I'll update to 1.3.7 at the same time.  We use pacparser in our go applications with my go-pacparser wrapper.  Thank you for the time and effort developing the library and test tool.